korge/src/commonMain/kotlin/com/soywiz/korge/particle/ParticleEmitterView.kt

```diff
@@ -100,8 +100,8 @@ class ParticleEmitterView(
 //                simulator.emitterPos.setTo(x, y)
 //            }
             if (dt > 0.milliseconds) {
-                val gx = globalX / stage!!.scaleX
-                val gy = globalY / stage!!.scaleY
+                val gx = globalX / (stage?.scaleX ?: 1.0)
+                val gy = globalY / (stage?.scaleY ?: 1.0)
```

Comment:
This seems from the previous PR, can you revert these changes here, so we can merge this PR?